core/config/locales/en.yml

```diff
@@ -305,10 +305,10 @@ en:
         cart_tax_country_iso: Tax Country for Empty Carts
       spree/store_credit:
         amount: Amount
-        amount_authorized: Amount Authorized
-        amount_credited: Amount Credited
-        amount_used: Amount Used
-        category_id: Credit Type
+        amount_authorized: Authorized
+        amount_credited: Credited
+        amount_used: Used
```

Comment:
I'm not sure these 3 keys are ok, since it could not be clear that they refer to the amount. 

Eg. 

```
Used cannot be greater than the credited amount
```

I don't get what we gain removing the term `Amount`

Reply:
You are right. I will change that